packages/client/src/omnitracking/postBatchTrackings.ts

```diff
@@ -24,7 +24,10 @@ import type { AxiosRequestConfig } from 'axios';
  *
  * @returns Promise object.
  */
-const postBatchTrackings = (data: Array<object>, config?: AxiosRequestConfig) =>
+const postBatchTrackings = (
+  data: Array<Record<string, unknown>>,
```

Comment:
There is an inconsistency between this type and the type defined for `data` in `postTrackings` client. The type for `data` in `postBatchTrackings` should be an array of elements of the same type defined for `data` in the `postTrackings` client. 

An alias should be defined to help with that, like this:

```js
type TrackingData = Record<string, unknown>;

// postBatchTrackings.ts
const postBatchTrackings = (
  data: Array<TrackingData>,
...

// postTrackings.ts
const postTrackings = (data: TrackingData, config?: AxiosRequestConfig) =>
...
```

Reply:
Fixed!